tests/cnf/ran/gitopsztp/tests/ztp-bios-day-zero.go

```diff
@@ -0,0 +1,127 @@
+package tests
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/bmh"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/hive"
+	"github.com/openshift-kni/eco-goinfra/pkg/nodes"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/gitopsztp/internal/tsparams"
+	. "github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/raninittools"
+	"github.com/openshift-kni/eco-gotests/tests/cnf/ran/internal/version"
+	"github.com/openshift-kni/eco-gotests/tests/internal/cluster"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("ZTP BIOS Configuration Tests", Label(tsparams.LabelBiosDayZeroTests), func() {
+	var (
+		spokeClusterName string
+		nodeNames        []string
+	)
+
+	// 75196 - Check if spoke has required BIOS setting values applied
+	It("Check if SNO spoke has required BIOS setting values applied", reportxml.ID("75196"), func() {
+		versionInRange, err := version.IsVersionStringInRange(RANConfig.ZTPVersion, "4.17", "")
+		Expect(err).ToNot(HaveOccurred(), "Failed to check if ZTP version is in range")
+
+		if !versionInRange {
+			Skip("ZTP BIOS configuration tests require ZTP version of least 4.17")
+		}
+
+		spokeClusterName, err = GetSpokeClusterName(HubAPIClient, Spoke1APIClient)
+		Expect(err).ToNot(HaveOccurred(), "Failed to get SNO cluster name")
+		By(fmt.Sprintf("Cluster name: %s", spokeClusterName))
```

Comment:
`By` should be left for separating chunks of code (and start with a lower case letter and a gerund), you can use a log instead
```go
glog.V(tsparams.LogLevel).Infof("cluster name: %s", spokeClusterName)
```